src/pytest_plugins/filler/filler.py

```diff
@@ -269,18 +270,53 @@ def pytest_report_header(config: pytest.Config):
 
 def pytest_report_teststatus(report, config: pytest.Config):
     """
-    Disable test session progress report if we're writing the JSON fixtures to
-    stdout to be read by a consume command on stdin. I.e., don't write this
-    type of output to the console:
+    This hook modifies the test results in pytest's terminal output.
 
+    We use this for:
+
+    1. To change the test results to display `FILLED` instead of `PASSED`.
+    2. To disable test session progress report if we're writing the JSON
+        fixtures to stdout to be read by a consume command on stdin. I.e.,
+        don't write this type of output to the console:
     ```text
     ...x...
     ```
     """
+    if report.when == "call" and report.passed:
+        return (report.outcome, ".", "FILLED")
     if is_output_stdout(config.getoption("output")):
         return report.outcome, "", report.outcome.upper()
 
 
+@pytest.hookimpl(hookwrapper=True, trylast=True)
+def pytest_terminal_summary(
+    terminalreporter: TerminalReporter, exitstatus: int, config: pytest.Config
+):
+    """
+    Modify pytest's terminal summary to emphasize that no tests were ran.
+
+    Emphasize that fixtures have only been filled; they must now be executed to
+    actually run the tests.
+    """
+    passed_display_text = "fixtures filled"
+    stats = terminalreporter.stats
+    if "passed" in stats:
+        terminalreporter._add_stats(passed_display_text, stats["passed"])
```

Comment:
This is deep into the pytest inner-workings. It might cause some trouble when updating pytest versions if these internal methods are renamed but should be ok.